tests/functional/conftest.py

```diff
@@ -129,15 +132,30 @@ def check_is_alive():
     Return a healthcheck function fixture for the GitLab container spinup.
     """
 
-    def _check(container: str, start_time: float) -> bool:
+    def _check(
+        *, container: str, start_time: float, docker_ip: str, docker_port: int
```

Comment:
We don't necessarily need to pass `docker_ip` and `docker_port` in here, we could directly use the `docker_ip` and `docker_services` fixtures in this fixture directly.

Then we again don't need to have the keywords discussion :rofl:

Reply:
e.g. you could have
```python
@pytest.fixture
def gitlab_url(docker_ip, docker_services):
    return f"http://{docker_ip}:{docker_services.port_for('gitlab-test')}"
```

Reply:
Thanks. Good idea. I did that and even more fixtures 😊